### PR TITLE
Add quay and aliyun creator

### PR DESCRIPTION
**Pull request by jgsqware**
Signed-off-by: Julien Garcia Gonzalez <julien@giantswarm.io>

Add the creation of Quay and Aliyun repository and notifications if needed.

### CODE REVIEW

File: aliyun/cr/api.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
+package cr
```
Comment (JosephSalisbury) [inline aliyun/cr/api.go]: cr?

Comment (jgsqware) [reply aliyun/cr/api.go]: aliyun refer Container Registry as `cr` that's why

you prefer `containerregistry` ?

Comment (JosephSalisbury) [reply aliyun/cr/api.go]: I think idiomatic golang would be `registry`. Single words for package names and all that.

**Comment by tfussell** [comment]
I think this is no longer necessary. I'm going to close it. Let me know if you disagree @jgsqware.

**Pull request closed without merging**